bin/build-release-ccd-definition.sh

```diff
@@ -6,7 +6,7 @@ environment=${1:-prod}
 excludeNonProdFiles=${2:-true}
 
 if [ ${environment} == preview ]; then
-   excludedFilenamePatterns="-e *-prod.json,*-SDO-HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*CUI-prod.json"
+   excludedFilenamePatterns="-e *-prod.json,*-GAR2GAspec-nonprod,*-SDO-HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*CUI-prod.json"
```

Comment:
I  think, in preview environment we dont need to exclude nonprod extension. Because we need nonprod files in nonprod environments (preview, local, demo,,,). We have to excluded nonprod files in prod environment. It exclusion should be included in last part of if and else statement "elif [ ${excludeNonProdFiles} == true ]; then ...." ATM in both condition as we are not live yet.